pkg/3scale/amp/component/backend.go

```diff
@@ -379,6 +388,15 @@ func (backend *Backend) buildBackendCommonEnv() []v1.EnvVar {
 		helper.EnvVarFromSecret("CONFIG_QUEUES_MASTER_NAME", BackendSecretBackendRedisSecretName, BackendSecretBackendRedisQueuesURLFieldName),
 		helper.EnvVarFromSecret("CONFIG_QUEUES_SENTINEL_HOSTS", BackendSecretBackendRedisSecretName, BackendSecretBackendRedisQueuesSentinelHostsFieldName),
 		helper.EnvVarFromSecret("CONFIG_QUEUES_SENTINEL_ROLE", BackendSecretBackendRedisSecretName, BackendSecretBackendRedisQueuesSentinelRoleFieldName),
+		// ACL
+		helper.EnvVarFromSecretOptional("CONFIG_REDIS_USERNAME", BackendSecretBackendRedisSecretName, BackendSecretBackendRedisStorageUsernameFieldName),
```

Comment:
should ACL fields be Optional? We have Authentication errors in worker and system

```
oc logs backend-worker-687b7c8c99-4cz4h
Defaulted container "backend-worker" out of: backend-worker, backend-redis-svc (init)
bundler: failed to load command: bin/3scale_backend_worker (bin/3scale_backend_worker)
/usr/share/gems/gems/protocol-redis-0.6.1/lib/protocol/redis/connection.rb:120:in `read_object': ERR AUTH <password> called without any password configured for the default user. Are you sure your configuration is correct? (Protocol::Redis::ServerError)
```
Maybe need change it to helper.EnvVarFromSecret (....) ?